x-pack/plugin/esql/qa/server/mixed-cluster/build.gradle

```diff
@@ -22,6 +22,7 @@ restResources {
 dependencies {
   javaRestTestImplementation project(xpackModule('esql:qa:testFixtures'))
   javaRestTestImplementation project(xpackModule('esql:qa:server'))
+  javaRestTestImplementation project(xpackModule('esql'))
```

Comment:
I don't see an obvious reason for this addition. Why did you need to add it?

Reply:
So the capabilities check can use the enum `JOIN_LOOKUP_V4` instead of a plan string, which I think would make it very likely if we changed the enum that we would get tests disabled that we did not want disabled. I saw Nhat added this to the MultiClusterSpecIT, so I did the same here.